.github/workflows/build.yaml

```diff
@@ -159,18 +167,9 @@ jobs:
             fi
           done
 
-      - name: 'Upload the repository to the bucket'
+      - name: 'Upload packages to GCS'
         run: |
           for arch in "x86_64" "aarch64"; do
-            # Don't cache the APKINDEX.
-            gcloud --quiet storage cp \
-                --cache-control=no-store \
-                "./packages/${arch}/APKINDEX.tar.gz" "gs://wolfi-production-registry-destination/os/${arch}/"
-
-            gcloud --quiet storage cp \
-                --cache-control=no-store \
-                "./packages/${arch}/APKINDEX.json" "gs://wolfi-production-registry-destination/os/${arch}/"
-
             # Only attempt to sign when *.apk's exist
```

Comment:
This comment seems to be out of place now? Was it always?

Reply:
Updated s/sign/upload/